var/spack/repos/builtin/packages/mgard/package.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Mgard(CMakePackage, CudaPackage):
+    """MGARD error bounded lossy compressor"""
+
+    homepage = "https://github.com/CODARcode/MGARD"
+    git = "https://github.com/robertu94/MGARD"
```

Comment:
In cases where you are maintaining a fork, can you mention the original repository in the docstring?

A short explanation of why the fork is used would be helpful (e.g. in the case of `nvcomp` that they went closed-source).

Reply:
@scheibelp please take a look at the most recent commit.